Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.11
+
+RUN apk update \
+  && apk add --no-cache ssmtp gettext libintl \
+  && mkdir /mail-config \
+  && mkdir /scripts \
+  && mkdir /etc/periodic/daily8am \
+  && mkdir /etc/periodic/daily1pm
+
+ADD mail-config /mail-config/
+ADD scripts /scripts/
+ADD cron_tasks_folder/daily8am/send-mail /etc/periodic/daily8am
+ADD cron_tasks_folder/daily1pm/send-mail /etc/periodic/daily1pm
+
```

Comment:
could use a `USER mail` or similar to avoid the `root` default

Reply:
... not sure I follow?

Reply:
By default, `alpine` runs everything as `root`:
```
$ docker run -it alpine:3.11 sh
/ # whoami
root
```
But stepping down to a less privileged user helps limiting which files can be overwritten by the scripts, peeking into possible volumes, escalation of privileges, etc
```
/ # grep mail /etc/passwd
mail:x:8:12:mail:/var/mail:/sbin/nologin
postmaster:x:14:12:postmaster:/var/mail:/sbin/nologin
vpopmail:x:89:89::/var/vpopmail:/sbin/nologin
```